apps/web/src/components/app/queue/QueueNextUp.tsx

```
@@ -40,7 +40,7 @@ export function QueueNextUp({
   );
 
   return (
-    <div className="space-y-3">
+    <div className="space-y-3" aria-label="Next Up">
```

Biome (2.4.4)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)